src/Components/DraggableResizer/DraggableResizerHandle.tsx

```diff
@@ -22,6 +34,7 @@ export interface DraggableResizerHandleProps extends StandardFunctionProps {
   /** Orientation of handle */
   orientation: Orientation
   handleBarStyle?: CSSProperties
+  handleBackgroundStyle?: CSSProperties
```

Comment:
I know you were just following the naming right above this line, but I think we should name this `backgroundStyle` handle gives an impression of a handler function.